README.rst

```diff
@@ -11,35 +11,35 @@ with interactive Jupyter notebooks in the cloud.
- - `Modeling of catalogue data <URL>`_
- - `Cosmic shear with Einstein ring simulations <URL>`_
+ - `Modeling a simple Einstein ring <https://github.com/lenstronomy/lenstronomy-tutorials/blob/main/Notebooks/LensModeling/modeling_a_simple_Einstein_ring.ipynb>`_
+ - `Modeling a doubly imaged quasar <https://github.com/lenstronomy/lenstronomy-tutorials/blob/main/Notebooks/LensModeling/modeling_a_doubly_imaged_quasar.ipynb>`_
+ - `Modeling a quadruply imaged quasar <https://github.com/lenstronomy/lenstronomy-tutorials/blob/main/Notebooks/LensModeling/modeling_a_quadruply_imaged_quasar.ipynb>`_
+ - `Modling multiple bands simultaneously <https://github.com/lenstronomy/lenstronomy-tutorials/blob/main/Notebooks/LensModeling/modeling_multiple_bands_simultaneously.ipynb>`_
+ - `Source reconstruction with shapelets <https://github.com/lenstronomy/lenstronomy-tutorials/blob/main/Notebooks/LensModeling/source_reconstruction_with_shapelets.ipynb>`_
+ - `Modeling of catalogue data <https://github.com/lenstronomy/lenstronomy-tutorials/blob/main/Notebooks/LensModeling/modelling_of_catalogue_data.ipynb>`_
+ - `Cosmic shear with Einstein ring simulations <https://github.com/lenstronomy/lenstronomy-tutorials/blob/main/Notebooks/LensModeling/cosmic_shear_with_Einstein_ring_simulations.ipynb>`_
 Numerics
- - `Solving lens equation <s>`_
- - `Lenstronomy numerics <s>`_
+ - `Solving lens equation <https://github.com/lenstronomy/lenstronomy-tutorials/blob/main/Notebooks/Numerics/solving_lens_equation_and_computing_flux_ratios.ipynb>`_
+ - `Lenstronomy numerics <https://github.com/lenstronomy/lenstronomy-tutorials/blob/main/Notebooks/Numerics/lenstronomy_numerics.ipynb>`_
 Simulations
- - `Simulation API <s>`_
- - `Skypy meets Lenstronomy <s>`_
-Galaxies
- - `Fitting galaxy light profile (like Galfit) <s>`_
- - `Quasar-host decomposition <s>`_
+ - `Simulation API <https://github.com/lenstronomy/lenstronomy-tutorials/blob/main/Notebooks/Simulations/simulation_api.ipynb>`_
+ - `Skypy meets Lenstronomy <https://github.com/lenstronomy/lenstronomy-tutorials/blob/main/Notebooks/Simulations/skypy_meets_lenstronomy.ipynb>`_
+Galaxy light fitting
+ - `Fitting galaxy light profile (like Galfit) <https://github.com/lenstronomy/lenstronomy-tutorials/blob/main/Notebooks/Galaxies/galfitting_with_lenstronomy.ipynb>`_
+ - `Quasar-host decomposition <https://github.com/lenstronomy/lenstronomy-tutorials/blob/main/Notebooks/Galaxies/quasar_host_galaxy_decomposition.ipynb>`_
 Dark matter substructure
- - `Substructure challenge - a simple example <s>`_
- - `Substructure Challenge - Mock production notebook <s>`_
+ - `Substructure challenge - a simple example <https://github.com/lenstronomy/lenstronomy-tutorials/blob/main/Notebooks/DMSubstructure/substructure_challenge_simple_example.ipynb>`_
+ - `Substructure Challenge - Mock production notebook <https://github.com/lenstronomy/lenstronomy-tutorials/blob/main/Notebooks/DMSubstructure/substructure_challenge_mock_production.ipynb>`_
```

Comment:
I would use lower case in challenge since it is not a specific challenge

Reply:
Done.